pkgs/by-name/au/authentik/package.nix

```diff
@@ -159,7 +159,8 @@ let
             --replace-fail 'dumb-init = "*"' "" \
             --replace-fail 'djangorestframework-guardian' 'djangorestframework-guardian2' \
             --replace-fail 'version = "4.9.4"' 'version = "*"' \
-            --replace-fail 'version = "<2"' 'version = "*"'
+            --replace-fail 'version = "<2"' 'version = "*"' \
+            --replace-fail '{ version = "5.0.0", platform = "darwin" }' '{ version = "*", platform = "darwin" }'
```

Comment:
Overwriting: https://github.com/goauthentik/authentik/blob/643c1f5bbf680e9faf724164d374360eec87d65e/pyproject.toml#L120

As the provided lxml dependency is 5.1.0.

Otherwise got:
```
┃        > Checking runtime dependencies for authentik-2024.2.2-py3-none-any.whl
┃        >   - lxml==5.0.0 not satisfied by version 5.1.0
```